### ISSUE TITLE
Unknown error while Tapping: General SOCKS server failure | Response text: ...

### ISSUE THREAD

**Issue by Zifirlk** [shamhi/HamsterKombatBot#2141]
не хочет кликать, тип прокси socks5. 
в других ботах прокся работает, дело не в ней.
![image](https://github.com/user-attachments/assets/847b8164-3d29-495e-b7d9-3232af27d292)

**Comment by cmeept** [comment]
Бот завис на мини гейм . Не важно есть прокси или нет

**Comment by Zifirlk** [comment]
насколько я понял, как пофиксить ты не знаешь? :(

**Comment by viplexa** [comment]
Да, после "Daily Mini Game already claimed" ничего не происходит, бот не работает

**Comment by kohutiv** [comment]
Ошибка в Taps. Отключил пока USE_TAPS=False. Работает

**Comment by darhanger** [comment]
Бот вообще ничего не делает. Не тапает, не улучшает, от слова совсем по 0.

**Comment by DenisSalnikoff** [comment]
Просто отключите пока тапы. Похоже разрабы игры что-то поменяли.

**Comment by prognt** [comment]
https://github.com/shamhi/HamsterKombatBot/pull/2142
